src/studio/nuxeo-studio/nuxeo-studio-release-notes/2018-06-11-studio-73-14.md

```diff
@@ -0,0 +1,88 @@
+  <fileset dir="${outdir}/marketplace/install/bundles"/>
+</nx:studioExtraction>
+```
+
+Here is an extract of the final result:
+
+```
+[...]
+    <nx:studioExtraction todir="${outdir}/marketplace">
+      <fileset dir="${outdir}/marketplace/install/bundles"/>
+    </nx:studioExtraction>
+
+    <zip destfile="${outdir}/${maven.project.artifactId}-${maven.project.version}.zip" basedir="${outdir}/marketplace"/>
+    <artifact:attach file="${outdir}/${maven.project.artifactId}-${maven.project.version}.zip" type="zip"/>
+  </target>
+</project>
+```
+
+Note that the above example assumes you have been generating your package using our recommended tool: [nuxeo-cli]({{page space='nxdoc' page='nuxeo-cli'}}). Otherwise you will need to adapt these instructions to your own use case.
+
+## Token Generation
+This release introduces the ability to generate personal authentication tokens from the Nuxeo Online Services portal.
+
+Tokens can be used as a replacement for a password, and can be revoked anytime. A few situations where they can be leveraged include:
+- In your continuous integration / continuous delivery chain, to have your software authenticate against our Maven private repository,
+- When you authenticate using a third party provider (Google or Okta) and need to use our command line tools with options that require a password to be provided, for instance to register an instance using nuxeoctl, link your Studio project to your Java development one in nuxeo-cli.
+
+To generate a token,
+- Login to <a target="_blank" href="https://connect.nuxeo.com">Nuxeo Online Services</a>
+- Visit the `My Tokens` tab
+- Create your token using the corresponding button and provide a clear name for it (ex: "CI Chain")
+![]({{file name='token-management.png'}} ?border=true,w=650)
+
+Your token will only be shown once, so make sure to save it. If you ever forget it or think it may have been compromised, you can revoke it anytime and generate a new one.
+
+## Suggestion Elements Generation Update
+Applies to LTS 2017 with Web UI.
+
+A new property has been introduced in Web UI's suggestion elements with LTS 2017 HF05, so that labels can always be shown for suggestions, not the id.
+
+Suggestion elements in your Studio Designer configuration will now include that additional property when you generate a layout or when you drag and drop a suggestion element (e.g. to suggest vocabulary entries, users or documents). As a consequence, this will require you to have HF05 installed on your instance for them to behave as expected.
+
+Existing layouts containing suggestion elements are kept as is, in order to prevent any change to your application's behavior without your consent. When Studio will detect elements using the old pattern, it will display a warning and suggest you to drag an drop the property in your layout again if you want to upgrade to the new format. Your configuration will keep working even if you don't, but you will not benefit from the latest upgrades.
```

Comment:
Existing layouts containing suggestion elements are kept as is, in order to prevent any change to your application's behavior without your consent. When Studio detects elements using the old pattern, it will display a warning and suggest that you drag and drop the property into your layout again if you want to upgrade to the new format. Your configuration will keep working even if you don't, but you will not benefit from the latest upgrades.